### ISSUE TITLE
Style of Bodydiode

### ISSUE THREAD

**Issue by AusterKordel** [circuitikz/circuitikz#102]
Is it possible to set the style of a bodydiode to strokediode? If I'm loading the package with the option 'strokediode' the bodydiode is shown as a 'emptydiod'. If I'm using 'fulldiode' it is working fine.

**Comment by Rmano** [comment]
Should be fixed with #144

**Comment by Rmano** [comment]
Seems to work --- if nobody is against it, I'll merge the fix in few hours:
```latex
\documentclass[border=10pt]{standalone}
\usepackage[siunitx]{circuitikz}
%\usepackage[siunitx]{circuitikzgit}
\begin{document}
\begin{circuitikz}[
    ]
    \begin{scope}[empty diodes]
        \draw (0,0) node[npn,bodydiode](npn){}++(2,0)node[pnp,
        bodydiode](npn){};
    \end{scope}
    \begin{scope}[stroke diodes]
        \draw (0,-2) node[nigbt,bodydiode](npn){}++(2,0)node[
        pigbt,bodydiode](npn){};
    \end{scope}
    \begin{scope}[full diodes]
        \draw (0,-4) node[nfet,bodydiode](npn){}++(2,0)node[
        pfet,bodydiode](npn){};
    \end{scope}
\end{circuitikz}
\end{document}
```
results in 
![image](https://user-images.githubusercontent.com/6414907/50045340-7550d380-0091-11e9-80f8-cb2fc3830721.png)
